gaffer-core/store/src/main/java/gaffer/store/schema/Schema.java

```diff
@@ -246,6 +289,15 @@ protected void addEntity(final String group, final SchemaEntityDefinition elemen
         super.addEntity(group, elementDef);
     }
 
+    @Override
+    public String toString() {
+        try {
+            return "Schema" + new String(toJson(true), "UTF-8");
```

Comment:
If encoding is a project level constant, this needs to be extracted and made available project-wide.